api/humiographql/alert.go

```diff
@@ -0,0 +1,39 @@
+package humiographql
+
+type Alert struct {
```

Comment:
Seperated the two structs as graphql alert conflict with our requirements for alerts